Formula/cflow.rb

```diff
@@ -1,9 +1,9 @@
 class Cflow < Formula
   desc "Generate call graphs from C code"
   homepage "https://www.gnu.org/software/cflow/"
-  url "https://ftp.gnu.org/gnu/cflow/cflow-1.5.tar.bz2"
+  url "https://ftp.gnu.org/gnu/cflow/cflow-1.6.tar.bz2"
   mirror "https://ftpmirror.gnu.org/cflow/cflow-1.5.tar.bz2"
```

Comment:
Given the mirror has not been updated, I have no idea WHY that passes our audit :(

Reply:
Huh. Glad you caught that. 😮